src/actions/tree.js

```diff
@@ -111,6 +124,15 @@ export const updateVisibleTipsAndBranchThicknesses = (
       visibilityToo: dispatchObj.visibilityToo
     });
 
+    /* update the history stack for zoom levels (if we are zooming) */
+    if (tree.idxOfFilteredRoot !== dispatchObj.idxOfInViewRootNode) {
```

Comment:
Should this line be 
```js
if (tree.idxOfInViewRootNode !== dispatchObj.idxOfInViewRootNode) {
```
Currently the revert button doesn't always work after clicking the "zoom to selected" button and I believe this is the cause.

Reply:
Yes! It should. Thanks :)